fieldservice_google_map/models/fsm_stage.py

```diff
@@ -0,0 +1,10 @@
+# Copyright (C) 2019 Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class FSMStage(models.Model):
+    _inherit = 'fsm.stage'
+
+    custom_color = fields.Char("Choose Color", default="red")
```

Comment:
In base fieldservice this field is default to hex codes.  The stages get loaded with hex code colors and user manually needs to update the custom color to get the icons showing on the google map.  Should there be a migration script or change in the data?

Reply:
Hello, I discussed this with Murtuza, and we're doing the following:

- For this module, keep the custom_color as is (hex chars) and remove the `web_widget_google_marker_icon_picker`dependency
- Create a second "connection" module, that will make the icon picker available in FSM if the `web_widget_google_marker_icon_picker` is installed.

This will require some enhancements on the oca/geospatial repo, so please keep an eye on the PRs landing there. They will be blocking progress here.

Reply:
@dreispt depends on #https://github.com/OCA/geospatial/pull/235